docs/infrastructure.md

```diff
@@ -1,10 +1,11 @@
-# SysAdmin
+These instructions are for a sysadmin looking to 
+deploy a new Neurobagel node locally in an institute or lab. 
```

Comment:
Maybe worth adding a line or two here before the next sentence that explicitly states we refer to this kind of deployment as a "node", and distinguish "public" from "local" nodes (to my understanding, the former could also be nodes deployed locally at an institute or lab, but they are data that can be openly queried).